src/components/SortButtons.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { TodoStatus } from '../types/SortTypes';
+import { Todo } from '../types/Todo';
+import classNames from 'classnames';
+
+interface Props {
+  todos: Todo[];
+  selectedStatus: TodoStatus;
+  onStatusChange: (status: TodoStatus) => void;
+  filteringTodosByActiveStatus: number;
+  TodoStatusRoutes: Record<TodoStatus, string>;
+}
+
+export const SortButtons: React.FC<Props> = ({
+  todos,
+  selectedStatus,
+  onStatusChange,
+  filteringTodosByActiveStatus,
+  TodoStatusRoutes,
+}) => {
+  if (todos.length > 0) {
+    return (
+      <footer className="todoapp__footer" data-cy="Footer">
+        <span className="todo-count" data-cy="TodosCounter">
+          {filteringTodosByActiveStatus} items left
+        </span>
+
+        <nav className="filter" data-cy="Filter">
+          {Object.keys(TodoStatusRoutes).map(status => (
```

Comment:
```suggestion
          {Object.values(TodoStatus).map(status => (
```
then status automatically will have the correct type